androidApp/src/main/kotlin/com/intive/picover/auth/intent/builder/SignInIntentBuilder.kt

```diff
@@ -1,22 +0,0 @@
-package com.intive.picover.auth.intent.builder
-
-import com.firebase.ui.auth.AuthUI
-import com.intive.picover.R
-import com.intive.picover.auth.intent.SignInIntent
-
-object SignInIntentBuilder {
-
-	fun build(authUi: AuthUI) =
-		SignInIntent(
-			authUi.createSignInIntentBuilder()
-				.setTheme(R.style.Theme_Picover)
```

Comment:
Theme was not adjusted to compose values so it's removed in shared implementation for simplicity. Otherwise we could pass it as a parameter of `rememberFirebaseAuthResultContract` or create a resource in android source set of shared module